### ISSUE TITLE
Failed to build with SVG support (librsvg-2.0)

### ISSUE THREAD

**Issue by dartmonkey876** [rdp/ffmpeg-windows-build-helpers#626]
Trying to build FFmpeg with SVG support, building for Win64 on Ubuntu 18.04 LTS

librsvg2-dev is installed, and pkg-config seems to see it

Error:
`
ERROR: librsvg-2.0 not found using pkg-config

If you think configure made a mistake, make sure you are using the latest
version from Git.  If the latest version fails, report the problem to the
ffmpeg-user@ffmpeg.org mailing list or IRC #ffmpeg on irc.libera.chat.
Include the log file "ffbuild/config.log" produced by configure as this will help
solve the problem.
failed configure ffmpeg_git_with_fdk_aac
`

Command used: ./cross_compile_ffmpeg.sh --enable-gpl=y --git-get-latest=y --build-ffmpeg-static=y --build-ffmpeg-shared=n --disable-nonfree=n --gcc-cpu-count=4

**Comment by dartmonkey876** [comment]
Here's my config.log if it'd help

[config.log](https://github.com/rdp/ffmpeg-windows-build-helpers/files/8310225/config.log)

**Comment by rdp** [comment]
Did you add something to enable rsvg?

**Comment by dartmonkey876** [comment]
I added "--enable-librsvg" to the config_options in build_ffmpeg()

**Comment by rdp** [comment]
Yeah you'd also need to add a section to build "librsvg" (the library) so
it can use it, hmm... :)
